core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -954,6 +954,37 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
+  /api/admin/workspaces/googleProject/{googleProjectId}:
```

Comment:
if we ever wanted an `/api/admin/workspaces/{workspaceNamespace}/{workspaceName}` API to get a workspace by its namespace+name, this route definition would mask it. It would think "googleProject" is a namespace and the `{googleProjectId}` is the workspace name.

I'm calling this out, but I don't think it's a huge problem - since these are admin APIs, we could always move them around later, and if we put `googleProject/{googleProjectId}` before `workspaceNamespace}/{workspaceName}` in AdminApiService, it would take precedence.

Do you think it's worth changing for this hypothetical?